src/plugins/behaviors/src/components/ground_targeted_aoe.rs

```diff
@@ -108,7 +108,7 @@ trait ColliderComponents {
 impl Instantiate for GroundTargetedAoeContact {
 	fn instantiate(&self, on: &mut EntityCommands, _: impl GetOrCreateAssets) -> Result<(), Error> {
 		let collider = self.collider_components()?;
-		let model = AssetModel::Path("models/sphere.glb");
+		let model = AssetModel::path("models/sphere.glb");
```

Comment:
Those 2 instances use it, because they deal with `String`. The `.path()` method was added for ease of use (so we do not have to write `.to_owned()` all the time). But the instances, where `::Path()` is retained are important, so that the project actually compiles.